docs/source/plugins/01_quick_start.rst

```diff
@@ -18,29 +18,21 @@ the official docs <https://www.python.org/downloads/>`_.
 
 .. note::
 
-    It is recommended that you install the |sdk| using a Python Virtual Environment.
-    For more details, see `Virtual Environments and Packages in Python documentation <https://docs.python.org/3/tutorial/venv.html>`_.
+    It is recommended that you install the |sdk| using the conda package manager.
 
-From a terminal, and inside a virtual environment, update pip:
+    You can it by adding a ``environment.yml`` to your project and declare the ``alfasim-sdk`` inside the ``pip`` dependencies.
+    For more details, see `Getting started with conda <https://conda.io/projects/conda/en/latest/user-guide/getting-started.html>`_.
 
-.. code-block:: console
+    And if you want to manage multiple repositories using conda, a further addition to your project could be the `conda devenv <https://github.com/ESSS/conda-devenv>`_ tool.
```

Comment:
I would remove this, probably will generate only noise to most users.

Reply:
Yeah, makes sense. It was removed.